app/code/Magento/QuoteGraphQl/Model/Cart/AddToCartHandlerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\QuoteGraphQl\Model\Cart;
+
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Quote\Model\Quote;
+
+/**
+ */
+interface AddToCartHandlerInterface
+{
+    /**
+     * @param CartInterface $cart
+     * @param ContextInterface $context
```

Comment:
Maybe?
    /**
     * @param Quote $cart
     * @param array $cartItemData
     * @return void
     */
I hope it helps.

Reply:
@TomashKhamlai yes, good spot. Thanks.